packages/api-gateway/test/auth.e2e-spec.ts

```diff
@@ -147,7 +170,11 @@ describe('Correct parameters to both generate auth key and jwt', () => {
           name: 'test-seed-project',
         },
       });
-    const apiKey = key.body['apiKey'];
+
+    let apiKey = '';
```

Comment:
I would just use the jest `toBeDefined` here, otherwise you'd be failing at the response code and it could be harder to debug